recipe/meta.yaml

```diff
@@ -30,13 +30,21 @@ source:
   sha256: 908742d8f3c6fdd6d1d6316a7b919b5c506474f9551f491aa7335cb4f50bffbd  # [win]
 
 build:
-  number: 1
+  number: 2
   skip: true  # [osx]
 
 test:
   commands:
     - test -d $PREFIX/lib/cmake                      # [linux]
     - test -d $PREFIX/targets/{{ target_name }}/lib  # [linux]
+    - if not exist %LIBRARY_LIB%\{{ target_name }}\cmake exit 1    # [win]
+    - if not exist %LIBRARY_INC%\targets\{{ target_name }} exit 1  # [win]
+    # Extracts the correct cccl version from Nvidia's cuda_cccl archive, and compares it against the version of the cccl conda package
+    - CCCL_VERSION_FILE="$PREFIX/targets/{{ target_name }}/lib/cmake/cccl/cccl-config-version.cmake"  # [linux]
+    - CCCL_VERSION_MAJOR=$(sed -n 's/set(CCCL_VERSION_MAJOR \([0-9]*\))/\1/p' ${CCCL_VERSION_FILE})  # [linux]
+    - CCCL_VERSION_MINOR=$(sed -n 's/set(CCCL_VERSION_MINOR \([0-9]*\))/\1/p' ${CCCL_VERSION_FILE})  # [linux]
+    - CCCL_VERSION_PATCH=$(sed -n 's/set(CCCL_VERSION_PATCH \([0-9]*\))/\1/p' ${CCCL_VERSION_FILE})  # [linux]
```

Comment:
As we discussed offline with @bdice, I'd suggest writing a `cmake -P` script that `include()`s the `cccl-config-version.cmake` file and prints out a machine-friendly version.

Reply:
I have this locally and will push in a moment. It is a bit cleaner.

Reply:
Updated in 34c0ddd0d88b1077c3d3da0da0bcc5ad81c21b2c.

Reply:
Thanks